third_party/hyperloglog/hyperloglog.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
Do we still need the new stuff in `third_party` since I don't think I see it used anywhere? (I think we use the hyperloglog module in `storage/stats`)

Reply:
removing